backend/inspirehep/records/utils.py

```diff
@@ -201,3 +201,11 @@ def _create_ticket_self_curation(record_control_number, record_revision_id):
         requestor=None,
         recid=record_control_number,
     )
+
+
+def get_changed_reference(old_record_version, new_record_version):
+    old_references = old_record_version.get("references", [])
+    new_references = new_record_version.get("references", [])
+    for old_ref, new_ref in zip(old_references, new_references):
+        if old_ref != new_ref:
+            return {"current_version": new_ref, "previous_version": old_ref}
```

Comment:
We should also serialise the records with the UI serializer

Reply:
yeah, good point 👍

Reply:
as we do https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/records/views.py#L85

Reply:
actually, this one will work for this case since we need just one ref https://github.com/inspirehep/inspirehep/blob/25852c0f31bb930a4454756e5e165d8ae72e3c78/backend/inspirehep/records/marshmallow/literature/common/reference_item.py#L21

Reply:
Sounds good!